docs/content/guides/dagster/parallel-computing-dask.mdx

```diff
@@ -0,0 +1,270 @@
+---
+title: Parallel computing with Dask on Dagster | Dagster Docs
+description: This guide illustrates how to parallel your compute with Dagster and Dask
+---
+
+# Parallel Computing with Dask on Dagster
+
+## What is Dask?
+
+[Dask](https://www.dask.org) is a flexible library for parallel computing in Python which can be a helpful tool when performing larger data engineering and machine learning tasks. This guide will review how to use [Dagster’s resources](https://docs.dagster.io/concepts/resources) to automate your Dask computations and make your pipelines faster. In this guide, you’ll learn how to:
+
+- Set up a Dask resources
+- Use Dask to speed up data assets
+- Use Dask’s machine learning tools to build a model
+
+---
+
+## Using Dagster’s resources to set up a Dask Cluster
+
+When using Dask with Dagster, you can use the `DaskResource` to set up a Dask client and simplify a pipeline that uses Dask across multiple assets. In this example, a local`dask.distributed` client is set up as a `ConfigurableResource`. Resources allow you to centralize your configuration in a single location, reducing redundant code in your assets.
+
+```python file=/guides/dagster/parallel-computing-dask/resources.py startafter=first_resource_start endbefore=first_resource_end
+from dask.distributed import Client
+
+# isort: skip_file
+from dagster import ConfigurableResource
+
+
+class DaskResource(ConfigurableResource):
+    def make_dask_cluster(self) -> Client:
+        client = Client()
+        return client
+```
+
+The `dask.distributed.LocalCluster` API allows customization of the cluster, which can be leveraged through Dagster’s `resource`. In the following example, configuration is setup at the resource level, which is then leveraged by Definitions and applied to all uses of the resource.
```

Comment:
I don't think "resource" should be monospaced,  because it doesn't correspond to a particular code symbol.